nengo_gui/components/slider.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, base_output, to_client=None, from_client=None):
         self.to_client = to_client
         self.from_client = from_client
 
-    def make_step(self, shape_in, shape_out, dt, rng):
+    def make_step(self, shape_in, shape_out, dt, rng, state=None):
```

Comment:
Yep, this looks good. I think you also need to change line 30 below to pass in `state`; you could do it in a `try/except` as a quick fix.

Reply:
Ah, good point...  Thank you!  I've added that via the try/except option (along with calling `make_state` to create the state in the first place)